source/utilities.tex

```diff
@@ -1941,7 +1941,8 @@
 template <size_t I, class... Types>
   constexpr tuple_element_t<I, tuple<Types...>>&& get(tuple<Types...>&& t) noexcept; // Note A
 template <size_t I, class... Types>
-  constexpr tuple_element_t<I, tuple<Types...>> const& get(const tuple<Types...>& t) noexcept; // Note B
+  constexpr tuple_element_t<I, tuple<Types...>> const&
```

Comment:
Just for visual harmony, would it be possible to break all three declarations in the same way?